packages/api/src/controllers/playback.ts

```diff
@@ -35,8 +36,11 @@ type PlaybackInfo = {
     live?: 0 | 1;
     playbackPolicy?: Asset["playbackPolicy"] | Stream["playbackPolicy"];
     source: {
-      hrn: "HLS (TS)" | "MP4";
-      type: "html5/application/vnd.apple.mpegurl" | "html5/video/mp4";
+      hrn: "HLS (TS)" | "MP4" | "WebRTC (VP8)";
+      type:
+        | "html5/application/vnd.apple.mpegurl"
+        | "html5/video/mp4"
+        | "html5/video/vp8";
```

Comment:
Wish there was something more specific we could put here, but couldn't find anything myself

Reply:
I just changed this - does H264 make more sense? I checked and this is what mist is using